manifests/olm-catalog/4.5/nfd.crd.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: nodefeaturediscoveries.nfd.openshift.io
+spec:
+  group: nfd.openshift.io
+  names:
+    kind: NodeFeatureDiscovery
+    listKind: NodeFeatureDiscoveryList
+    plural: nodefeaturediscoveries
+    singular: nodefeaturediscovery
+  scope: Namespaced
```

Comment:
Just popped into my mind should this be namespaced or not? Does it make sense to have multiple NFD operator deployments in the same cluster, maybe it does, wdyt?

Reply:
I guess is a very corner case, but still possible for some admins to have the same operator constrained to diff namespaces, depending on teams or something, is a very corner case, but still can think on someone who might needed it in a mid term future

Reply:
Let's keep it as it is